lldb/include/lldb/ValueObject/DILEval.h

```diff
@@ -54,6 +54,15 @@ class Interpreter : Visitor {
   Visit(const ArraySubscriptNode *node) override;
   llvm::Expected<lldb::ValueObjectSP>
   Visit(const BitFieldExtractionNode *node) override;
+  llvm::Expected<lldb::ValueObjectSP>
+  Visit(const IntegerLiteralNode *node) override;
+  llvm::Expected<lldb::ValueObjectSP>
+  Visit(const FloatLiteralNode *node) override;
+
+  llvm::Expected<CompilerType>
```

Comment:
Should we leave a comment saying that this is not copied from any particular integer promotion rules, and specific to the DIL DSL?